runtime/compiler/env/J9ClassEnv.cpp

```diff
@@ -1038,3 +1038,22 @@ J9::ClassEnv::flattenedArrayElementSize(TR::Compilation *comp, TR_OpaqueClassBlo
       return vm->internalVMFunctions->arrayElementSize((J9ArrayClass*)self()->convertClassOffsetToClassPtr(arrayClass));
       }
    }
+
+j9object_t*
+J9::ClassEnv::getDefaultValueSlotAddress(TR::Compilation *comp, TR_OpaqueClassBlock *clazz)
+   {
+#if defined(J9VM_OPT_JITSERVER)
+   if (auto stream = TR::CompilationInfo::getStream())
```

Comment:
Suggest using `comp->getStream()` instead of `TR::CompilationInfo::getStream()` to avoid thread local storage access.

Reply:
Addressed in 1459551